activestorage/app/jobs/active_storage/transform_job.rb

```diff
@@ -3,10 +3,10 @@
 class ActiveStorage::TransformJob < ActiveStorage::BaseJob
   queue_as { ActiveStorage.queues[:transform] }
 
-  discard_on ActiveRecord::RecordNotFound, ActiveStorage::UnrepresentableError
+  discard_on ActiveRecord::RecordNotFound
```

Comment:
`TransformJob` is only called from one place and should not be called with an unrepresentable blob. Rather than discarding, I think it should raise an exception if someone were to break this.